prime-dotnet-webapi/Services/SubmissionService.cs

```diff
@@ -304,6 +305,29 @@ private async Task RerunRulesAsync(Enrollee enrollee)
             await _enrolleeService.RemoveNotificationsAsync(enrollee.Id);
         }
 
+        public async Task RerunRulesAsync()
+        {
+            var enrollees = GetBaseQueryForEnrolleeApplicationRules()
+                .Where(e => e.Adjudicator == null)
+                .Where(e => e.CurrentStatus.StatusCode == (int)StatusType.UnderReview)
+                // TODO: Don't hard-code StatusReasonCodes
+                .Where(e => e.CurrentStatus.EnrolmentStatusReasons.Any(esr => esr.StatusReasonCode > 2 && esr.StatusReasonCode < 8))
```

Comment:
I would prefer a list of applicable status reason codes with a `.Contains()` rather than doing a range operation here. Its confusing what the intent of the code is.

Reply:
Also, you don't need the magic numbers, we have `StatusReasonType`

Reply:
Apologies, you may have explained before, but is there a reason the enum is named `StatusReasonType` rather than `StatusReasonCode`?

Reply:
It was the convention near the start of the app's development. I think calling them codes is a bit nicer in most cases but there are a few times it can be confusing. Also, `StatusCodes` is unfortunately already an enum (for http status codes like 200Ok)